src/stories/Library/opening-hours/OpeningHoursWeekList.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import OpeningHoursDayEntry from "./OpeningHoursDayEntry";
+import { GroupedOpeningHours } from "./OpeningHoursExampleData";
+
+type OpeningHoursWeekListProps = {
+  groupedOpeningHours: GroupedOpeningHours;
+};
+
+const OpeningHoursWeekList: React.FC<OpeningHoursWeekListProps> = ({
+  groupedOpeningHours,
+}) => {
+  return (
+    <ul className="opening-hours__content">
+      {groupedOpeningHours.map(({ day, date, openingHourEntries }) => {
+        const formattedDateForDisplay = new Date(date).toLocaleDateString(
+          "da-DK",
+          {
+            day: "numeric",
+            month: "2-digit",
+          }
+        );
+
+        return (
+          <li key={date} className="opening-hours__row">
+            <h3 className="opening-hours__individual-day">{`${day}: d. ${formattedDateForDisplay}`}</h3>
+            {openingHourEntries.length > 0 ? (
+              <ul>
```

Comment:
Consider if these lists are actually ordered (`<ol>` instead of `<ul>`).

In general I appreciate the use of list elements here.

Reply:
I think it makes sense to used OL since they atleast the weeks follow an specific sequence. 👍  Will for "categories" aswell.